src/styles/main.scss

```diff
@@ -1,3 +1,345 @@
+    margin-left: --base-shift;
+    margin-right: --base-shift;
+    
+    
+  }
+
+  .field__li {
+    margin-right: 50px;
+  
+  }
+}
+
+@media screen and (width: 1024px) {
+  .field__main {
+    margin-left: --base-shift;
+    margin-right: --base-shift;
+   
+  }
+
+  .field__li {
+    margin-right: 50px;
+  }
+} */
+
+/* #region header */
+.header {
+  width: 100%;
+  font-size: 12px;
+  /* box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05); */
+}
+
+.header__inner-container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  position: relative;
+  /* bottom: 1px; */
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  padding: 22.5px 0;
+  margin-right: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+  margin-right: 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__refer {
+  text-decoration: none;
+  color: #000;
+  position: relative;
+  padding: 22.5px 0;
+}
+
+.nav__refer:visited {
+  color: #00acdc;
+}
+
+.nav__refer:hover {
+  color: #00acdc;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__item:first-child {
+  margin: 0 10px 0 0;
+}
+
+.nav__item:last-child {
+  margin: 0 0 0 10px;
+}
+
+.is-active:link {
+  color: #00acdc;
+}
+
+.nav__refer:active::after,
+.nav__link :first-child .nav__refer::after {
+  display: block;
+  position: absolute;
+  top: 56px;
+  content: "";
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+}
+/* #endregion */
+
+/* #region main */
+
+.field {
+  width: var(--field-width);
+}
+
+.field__main {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 0;
+  margin-top: 0;
+  margin-left: var(--base-shift);
+  margin-right: var(--base-shift);
+
+}
+
+.field__li {
+  list-style: none;
+  width: 198px;
+  height: 406px;
+  /* margin-right: 4.17%; */
+  margin-bottom: 48px;
+  margin-top: 51px;
+  justify-content: center;
+  /* flex-grow: 1; */
+  /* flex-basis: auto; */
+  margin-right: 50px;
+  
+}
+
+.field__li:nth-child(4n) {
+  margin-right: 0;
+}
+
+.field__li:nth-child(n + 5) {
+  margin-top: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  flex-direction: row;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  list-style: none;
+  width: 16px;
+  height: 16px;
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+/* #region card */
+
+/* .card:nth-last-child(-n + 4) {
+  margin-right: 48;
+} */
+
+.card {
+  /* box-sizing: border-box; */
+  display: flex;
+  flex-direction: column;
```

Comment:
it's better to not set flex here. just use display: block and set 200 px width.  Each `div` starts from a new line, because it has 100% width